osu.Game.Tests/Visual/TestCaseBeatmapInfoWedge.cs

```diff
@@ -126,6 +127,26 @@ private void testNullBeatmap()
             AddAssert("check no infolabels", () => !infoWedge.Info.InfoLabelContainer.Children.Any());
         }
 
+        private void testAsyncLoading()
+        {
+            RulesetInfo firstRuleset = rulesets.AvailableRulesets.ElementAt(0); // Should be osu standard
+            RulesetInfo secondRuleset = rulesets.AvailableRulesets.ElementAt(1); // Should be taiko
+
+            var firstBeatmap = createTestBeatmap(firstRuleset, 25000);
+            var secondBeatmap = createTestBeatmap(secondRuleset, 1);
+
+            BeatmapInfoWedge.BufferedWedgeInfo infoBefore = null;
+            AddStep("select two beatmaps", () =>
+            {
+                infoBefore = infoWedge.Info;
+                infoWedge.UpdateBeatmap(beatmap.Value = new TestWorkingBeatmap(firstBeatmap));
+                infoWedge.UpdateBeatmap(beatmap.Value = new TestWorkingBeatmap(secondBeatmap));
+            });
+            AddUntilStep(() => infoWedge.Info != infoBefore, "wait for load");
+            AddWaitStep(3);
```

Comment:
I loaded two beatmaps, I would like to know that the other beatmap loaded too

Reply:
Yeah I can't find a way to actually make sure both beatmaps loaded. It would require a new variable that is updated even if the newly loaded BufferedWedgeInfo is no longer needed by BeatmapInfoWedge.

Reply:
We don't need to test that both got loaded. That would be testing `LoadComponentAsync()`, which is not for this component to test.

This line should be removed.